benchmarks/decoders/benchmark_decoders_library.py

```diff
@@ -518,6 +518,14 @@ def plot_data(df_data, plot_path):
         for col in range(video_type_combinations[unique_videos[row]], max_combinations):
             fig.delaxes(axes[row, col])
 
+    plt.gcf().text(
```

Comment:
I would prefer top left instead of bottom left, but I'll leave it to you

Reply:
Also, is_cuda_available isn't that useful. It would be better to get the GPU version

Reply:
Done. Honestly, neither look "good". And I realized we need to do either top or bottom; that will be robust to changing the number of videos.